src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -22,6 +22,10 @@ public static class SchemaVersionConstants
         public const int AddPrimaryKeyForResourceTable = (int)SchemaVersion.V25;
         public const int AddTextHashForStringSearchParameterVersion = (int)SchemaVersion.V26;
```

Comment:
We should remove this since we are adding schemaVersion 26 with new name and use that